package.json

```diff
@@ -112,21 +113,20 @@
     "prop-types": "15.5.10",
     "raven-js": "3.15.0",
     "raw-loader": "0.5.1",
-    "react": "15.1.0",
-    "react-addons-css-transition-group": "15.1.0",
-    "react-addons-test-utils": "15.1.0",
-    "react-document-title": "2.0.1",
-    "react-dom": "15.1.0",
+    "react": "15.5.4",
+    "react-addons-css-transition-group": "15.5.2",
+    "react-addons-test-utils": "15.5.1",
+    "react-document-title": "2.0.3",
+    "react-dom": "15.5.4",
     "react-hot-loader": "1.3.0",
     "react-imageloader": "2.1.0",
```

Comment:
it was throwing warnings and errors with the upgrade of React and the repo is pretty much dead. https://github.com/hzdg/react-imageloader